common/components/filter/_filter.scss

```diff
@@ -75,3 +75,46 @@
 .app-filter__label {
   text-decoration: underline;
 }
+
+.app-filter--stacked {
+  .app-filter__list {
```

Comment:
So another way of doing this, to avoid having to nest all these overrides under the parent modifier is to use [parent selector](https://sass-lang.com/documentation/style-rules/parent-selector):

```scss
.app-filter__list {
  ...

  .app-filter--stacked & {
    flex-direction: column;
  }
}
```

The main benefit of this organisation is that you can see the way a specific element changes based on parent modifiers. So if you change that element, it's clearer that you may affect behaviour for other modifiers.

Not a big deal for now, but one to think about in future when doing similar changes.

Reply:
Ok, thank you for pointing this out.